src/marks/auto.d.ts

```diff
@@ -3,26 +3,74 @@ import type {CompoundMark, Data} from "../mark.js";
 import type {Reducer} from "../reducer.js";
 import type {BinOptions} from "../transforms/bin.js";
 
+// TODO: should jsdoc go here or inside interface?
+type MarkType = "dot" | "line" | "area" | "rule" | "bar";
+
 export interface AutoOptions {
+  /**
+   * The horizontal spatial encoding. If *y* is set without *x*, the *x* channel
+   * will default to the **count** reducer to produce a histogram; you can
```

Comment:
```suggestion
   * The horizontal spatial encoding. If **y** is set without **x**, the **x** channel
   * will default to the *count* reducer to produce a histogram; you can
```

channel names in boldface, string option in italics